sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -190,42 +191,13 @@ object AkkaGrpcPlugin extends AutoPlugin {
     else generators
   }
 
-  // this transforms the Akka gRPC API generators to the right protocbridge type
-  def toGenerator(
-      codeGenerator: akka.grpc.gen.CodeGenerator,
-      scalaBinaryVersion: ScalaBinaryVersion,
-      logger: GenLogger): protocbridge.Generator = {
-    // This matches the sbt binary version (2.12)
-    val codegenScalaBinaryVersion = CrossVersion.binaryScalaVersion(BuildInfo.scalaVersion)
-    protocbridge.SandboxedJvmGenerator(
-      codeGenerator.name,
-      protocbridge.Artifact(BuildInfo.organization, s"${BuildInfo.name}_$codegenScalaBinaryVersion", BuildInfo.version),
-      codeGenerator.suggestedDependencies(scalaBinaryVersion),
-      new ProtocBridgeSbtPluginCodeGenerator(_, codeGenerator.getClass.getName, logger))
-  }
-
-  /**
-   * Uses reflection to load a [[akka.grpc.gen.CodeGenerator]] and turns it into protocbridge required type.
-   */
-  private[akka] class ProtocBridgeSbtPluginCodeGenerator(classLoader: ClassLoader, className: String, logger: GenLogger)
-      extends protocbridge.ProtocCodeGenerator {
-    val genClass = classLoader.loadClass(className)
-    val module = genClass.getField("MODULE$").get(null)
-    private val reflectiveLogger: Object =
-      classLoader
-        .loadClass("akka.grpc.gen.ReflectiveLogger")
-        .asInstanceOf[Class[Object]]
-        .getConstructor(classOf[Object])
-        .newInstance(logger)
-
-    private val runMethods =
-      module.getClass.getMethods
-        .find(m => m.getName == "run" && m.getParameterTypes()(0) == classOf[Array[Byte]])
-        .getOrElse(throw new RuntimeException("Could not find 'run' method that takes an Array[Byte]"))
-
-    override def run(request: Array[Byte]): Array[Byte] =
-      runMethods.invoke(module, request, reflectiveLogger).asInstanceOf[Array[Byte]]
-    override def toString = s"ProtocBridgeSbtPluginCodeGenerator(${className})"
-  }
+  def sandboxedGenerator(codeGenerator: akka.grpc.gen.CodeGenerator): Def.Initialize[protocbridge.Generator] =
+    Def.setting {
+      GeneratorBridge.sandboxedGenerator(codeGenerator, ScalaBinaryVersion(scalaBinaryVersion.value), generatorLogger)
+    }
 
+  def jvmGenerator(codeGenerator: akka.grpc.gen.CodeGenerator): Def.Initialize[protocbridge.Generator] =
```

Comment:
It might be a bias from knowing the sbt-protoc wording, but I find it slightly confusing to use `jvm` alone, as `sandboxedGenerator` also exposes a generator defined in a JVM bytecode. Not sure it's better, but to better emphasize the difference with `sandboxedGenerator`, what about 
```suggestion
  def sharedJvmGenerator(codeGenerator: akka.grpc.gen.CodeGenerator): Def.Initialize[protocbridge.Generator] =
```

Reply:
Actually the wording is the same for that https://github.com/scalapb/protoc-bridge/blob/f0e09cecfdef9f352b17acd94b9d87435f0633b1/bridge/src/main/scala/protocbridge/Generator.scala#L28-L32, but the JVM is also present in the sandboxed version.

Reply:
I guess the 'jvm' is a bit silly here because we don't support non-JVM generators (if you want to use those, you can just add them to PB.targets).

I agree `jvmGenerator` is a bit obscure - perhaps `plainGenerator` would be better? Or `unsandboxed`? I generally dislike negations in names, but in this case it might be justified since it is a 'somewhat dangerous'. WDYT?

Reply:
:+1: